terraform/modules/tf_stream_alert_app/main.tf

```diff
@@ -0,0 +1,69 @@
+// Lambda Function: StreamAlert App
+// Naming convention: '<prefix>_<cluster>_<service>_<app_name>_app'
+resource "aws_lambda_function" "stream_alert_app" {
+  function_name = "${var.function_prefix}_app"
+  description   = "StreamAlert App for ${var.type}"
+  runtime       = "python2.7"
+  role          = "${aws_iam_role.stream_alert_app_role.arn}"
+  handler       = "${var.stream_alert_apps_config["handler"]}"
+  memory_size   = "${var.app_memory}"
+  timeout       = "${var.app_timeout}"
+  s3_bucket     = "${var.stream_alert_apps_config["source_bucket"]}"
+  s3_key        = "${var.stream_alert_apps_config["source_object_key"]}"
+
+  environment {
+    variables = {
+      CLUSTER      = "${var.cluster}"
+      LOGGER_LEVEL = "${var.log_level}"
+    }
+  }
+
+  dead_letter_config {
+    target_arn = "arn:aws:sns:${var.region}:${var.account_id}:${var.monitoring_sns_topic}"
+  }
+
+  tags {
+    Name = "App:${var.type}"
```

Comment:
This should still have a StreamAlert name tag similar to the others for billing purpose.  Additionally, you can add a new App key

Reply:
Oh nice - I didn't realize that was the purpose. Would that just look similar to the following?
```
tags {
    Name  = "StreamAlert"
    App   = ${var.type}
}
```

Reply:
Yep